examples/kind/.github/generate-reports.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
is this exactly the same as the other file?

Reply:
yes conformance.sh and generate-reports.sh in kind and gce directory are the same